nx_main/nx_power.c

```diff
@@ -0,0 +1,37 @@
+#include <switch.h>
+#include "../common/common.h"
+
+static bool psmInitialized;
+
+void powerGetDetails(uint32_t *batteryCharge, bool *isCharging) {
```

Comment:
This should return a bool indicating whether the the output is valid (when psmInitialized block runs), caller shouldn't display info in that case.

Reply:
return false